src/components/transactions/transactionDetailViewRow.js

```diff
@@ -6,7 +6,7 @@ import styles from './transactionDetailView.css';
 class TransactionDetailViewRow extends React.Component {
   render() {
     const { children, shouldShow } = this.props;
-    return ((shouldShow === null || shouldShow === false) ? null :
+    return ((shouldShow === null || shouldShow === false || shouldShow === '') ? null :
```

Comment:
I think this does the same:
```
(!shouldShow)
```

Reply:
`! shouldShow` will also cover `undefined` and We cant do it